app/ui/src/app/import-export/import-export.component.scss

```diff
@@ -0,0 +1,24 @@
+:host {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  overflow: visible;
+}
+
+ul {
```

Comment:
You do not need this `ul` styling anymore. Jas add the `syn-tabs` class to the navbar and some additional classnames to the listitems and you're all sel. 

Double check the example in the Monitor metrics for reference :)